zeebe/util/src/main/java/io/camunda/zeebe/util/collection/ReusableObjectList.java

```diff
@@ -75,6 +75,17 @@ public T poll() {
     return null;
   }
 
+  public T peek() {
```

Comment:
🔧 I think this can be improved:

```
public T peek() {
    for (ReusableElement element : elements) {
        if (element != null && element.isSet()) {
            return element.getElement();
        }
    }
    return null;
}
```